modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1101,6 +1101,62 @@ definition {
 		}
 	}
 
+	@description = "LPS-156704 - Verify that the user is not able to order the object entry's table by the aggregation field column"
+	@priority = "4"
+	test CanNotOrderByAggregationFieldColumn {
+		task ("Create Custom Object and Custom Relationship with Account (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161891",
+				objectName = "CustomObject161891",
+				pluralLabelName = "Custom Objects 161891");
```

Comment:
Use the ticket number correct to add the Custom Object - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)